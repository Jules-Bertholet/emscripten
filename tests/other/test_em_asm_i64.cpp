#include <stdint.h>
#include <stdio.h>
#include <limits.h>
#include <emscripten/em_asm.h>

int main() {
  int64_t num = (2LL << 55) | 42LL;
  printf("native = %lld\n", num);

  EM_ASM({
    console.log("js     = " + $0);
  }, num);
}
