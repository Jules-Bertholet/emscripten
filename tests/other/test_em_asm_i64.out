native = 72057594037927978
js     = 72057594037927978
